src/gui/static/src/app/components/pages/force-skywallet-wipe/force-skywallet-wipe.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163601)
```suggestion
import { Router } from "@angular/router";
```